java/cuvs-java/src/main/java/com/nvidia/cuvs/CagraQuery.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Arrays;
 import java.util.List;
```

Comment:
This can be removed now, I think.  You've removed it from `BruteForceQuery.java`.
```suggestion
```